core/domain/html_cleaner.py

```diff
@@ -26,24 +26,26 @@
 from core.domain import rte_component_registry
 
 
-def filter_a(name, value):
+def filter_a(tag, name, value):
```

Comment:
If you are passing in 'a', then this is now a generic filter_attributes function and should be renamed accordingly. The description should be updated to be more generic, too.

Reply:
Hi @seanlip and @apb7, 
I read the details [here](https://github.com/mozilla/bleach/pull/263) as suggested by you.
we can rename it to `filter_tag ` or keep it unchanged as only 'a' attributes will be checked.

Reply:
Looking at the [docs](https://bleach.readthedocs.io/en/latest/clean.html#using-functions), I think you can keep it as filter_a but change `tag` to `unused_tag` and leave the method as it was (like in the second example in the docs). Or you could do something similar to the first example in the docs and pass in a function, in which case you would change this to filter_tag.

Probably the first one is better.

Reply:
Yes we can go with the first one and rename the tag to unused_tag, but if we have no mention of the function parameter (tag) throughout the function linter may raise an “unused parameter tag” error.
thats why we used  if tag=‘a’ just to mention/use all function the  parameter

In the example also, they have not used all the function parameters, i.e not used “tag” anywhere inside the function

Reply:
The linter should ignore arguments that start with `unused_` for the purposes of the check that you mentioned. I suggest trying it.

Reply:
> Hi @seanlip & @apb7,
> I've updated it learning from the examples in the doc you mentioned.
> I've renamed it to filter_tag and it made available/callable for all the tags.
> PTAL Thanks!

I think you might have misread my comment. If you're going with the first approach, please keep it as filter_a.

(In the second approach, you would pass in a function rather than a dict for the 'attrs' argument. But I think this is messier and a dict would be simpler.)

Reply:
Hi @seanlip, Updated PTAL  thanks!